StorageUtility/Linux/StorageUtility.cpp

```diff
@@ -210,4 +211,11 @@ void CloseDeviceHandle(DeviceHandle& Handle)
     }
 }
 
+eErrorCode GetPhysicalDiskNumber(DeviceHandle Handle, U32& PhysicalDiskNumber)
```

Comment:
About this function, I think we are developing VTStor to cross platforms, if we want get some information by Handle (on Window) then VTStor on Linux cant not implement. Please remember